data/strings/strings.txt

```diff
@@ -26156,90 +26156,92 @@
     tr = USB kablosunu çıkarın
     zh-Hant = 拔除 USB 線
 
-  [enable_screen_sleep]
+  [enable_keep_screen_on]
     tags = android
-    en = Allow the screen to sleep
-    af = Laat skerm toe om te slaap
-    ar = اسمح للشاشة بالنوم
-    be = Дазволіць экрану засынаць
-    bg = Позволяване на екрана да заспи
-    ca = Permet apagar la pantalla
-    cs = Nechte obrazovku spát
-    da = Lad skærmen sove
-    de = Bildschirm schlafen lassen
-    el = Αφήστε την οθόνη να κοιμηθεί
-    es = Permitir que la pantalla duerma
-    et = Luba ekraanisäästja
-    eu = Utzi pantailari lo egiten
-    fa = اجازه دهید صفحه نمایش بخوابد
-    fi = Salli näytön lepotila
-    fr = Autoriser l'écran à se mettre en veille
-    he = אפשר למסך לישון
-    hu = Hagyja aludni a képernyőt
-    id = Izinkan layar untuk tidur
-    it = Consenti allo schermo di dormire
-    ja = 画面をスリープ状態にする
-    ko = 화면 절전 모드 허용
-    lt = Leisti ekranui užmigti
-    mr = स्क्रीनचे निद्रा कार्य चालू करा
-    nb = La skjermen sove
-    nl = Laat het scherm slapen
-    pl = Pozwól na wygaszanie ekranu
-    pt = Permitir que o ecrã desligue
-    pt-BR = Permitir que a tela hiberne
-    ro = Permite ecranului să se stingă
-    ru = Разрешить экрану спать
-    sk = Nechajte obrazovku spať
-    sv = Låt skärmen sova
-    th = อนุญาตให้หน้าจอเข้าสู่โหมดสลีป
-    tr = Ekranın kapanmasına izin ver
-    uk = Дозволити екрану вимкнутись
-    vi = Cho phép màn hình ở chế độ ngủ
-    zh-Hans = 允许屏幕进入休眠状态
-    zh-Hant = 允许螢幕進入休眠狀態
-
-  [enable_screen_sleep_description]
+    en = Keep the screen on
```

Comment:
@rtsisyk please make a follow-up fix.

tag = android
comment = Setting in preferences to completely disable screen dimming and auto device lock